scripts/tests/test_box_migrate_to_external_account.py

```diff
@@ -0,0 +1,86 @@
+from nose.tools import *
+
+from scripts.box.migrate_to_external_account import do_migration, get_targets
+
+from framework.auth import Auth
+
+from tests.base import OsfTestCase
+from tests.factories import ProjectFactory, UserFactory
+
+from website.oauth.models import ExternalAccount
+from website.addons.box.model import BoxUserSettings, BoxOAuthSettings
+from website.addons.box.tests.factories import BoxOAuthSettingsFactory, BoxUserSettingsFactory
+
+
+class TestBoxMigration(OsfTestCase):
+    # Note: BoxUserSettings.user_settings has to be changed to foreign_user_settings (model and mongo). See migration instructions
+
+    def setUp(self):
+        super(TestBoxMigration, self).setUp()
+        self.user1 = UserFactory()
+        self.user1_settings = BoxUserSettingsFactory(owner=self.user1)
+        self.user1_settings.oauth_settings = BoxOAuthSettingsFactory()
+        self.user1_settings.save()
+
+    def tearDown(self):
+        super(TestBoxMigration, self).tearDown()
+        BoxUserSettings.remove()
+        ExternalAccount.remove()
```

Comment:
No. If these lines aren't run, objects persist between test cases and break things.